impeller/display_list/display_list_dispatcher.cc

```diff
@@ -490,8 +491,22 @@ static Path ToPath(const SkPath& path) {
         break;
     }
   } while (verb != SkPath::Verb::kDone_Verb);
-  // TODO: Convert fill types.
-  return builder.TakePath();
+
+  FillType fill_type;
+  switch (path.getFillType()) {
+    case SkPathFillType::kWinding:
+      fill_type = FillType::kNonZero;
+      break;
+    case SkPathFillType::kEvenOdd:
+      fill_type = FillType::kOdd;
+      break;
+    case SkPathFillType::kInverseWinding:
+    case SkPathFillType::kInverseEvenOdd:
+      UNIMPLEMENTED;
```

Comment:
Is this something that we'll eventually need? If so let's add a TODO and file an issue.

Reply:
Done: https://github.com/flutter/flutter/issues/104848

Reply:
FWIW, Flutter does not expose these methods - https://api.flutter.dev/flutter/dart-ui/PathFillType.html only does evenOdd and nonZero.

Reply:
(and nonZero == kWinding)